### ISSUE TITLE
Style of tags of tech stack in ProjectList [style]: 

### ISSUE THREAD

**Issue by anonymgit** [priyankarpal/ProjectsHut#639]
### What's the style idea?

> **IDEA** : to changes the style of tech stack tags in ProjectList page.
### previous
![image](https://github.com/priyankarpal/ProjectsHut/assets/62369735/6adacc88-0dab-4c6f-92fc-3b2947815bd3)

### After changes it will look like
![image](https://github.com/priyankarpal/ProjectsHut/assets/62369735/5fba0c57-1156-471e-8477-eb579d8b3bc5)

### Add screenshots

screenshots  **Daylight**
![image](https://github.com/priyankarpal/ProjectsHut/assets/62369735/be2737dd-45ab-482f-98b0-6b84effb99f8)
screenshots **Dark mode**
![image](https://github.com/priyankarpal/ProjectsHut/assets/62369735/81c6cf5b-1931-427f-aec3-21421f010576)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by priyankarpal** [comment]
yes. agreed. but you  have to change the bg & text color to the theme colors

assigned to you

**Comment by anonymgit** [comment]
> yes. agreed. but you have to change the bg & text colour to the theme colours
> 
> assigned to you
> 
> are you in GSSoC'23?

> yes "I'm in GSSOC'23"

**Comment by priyankarpal** [comment]
Completed. Thanks for your contributions 🎉 ❤️
